x/docs/md/cli/windows.md

```diff
@@ -1,80 +1,110 @@
 
-#### Using PowerShell
-Windows ships with a powerful scripting language called [PowerShell](https://en.wikipedia.org/wiki/Windows_PowerShell).
+If you already have the Chocolatey Package Manager installed, feel free to skip to the [Exercism CLI installation](#install-exercism-cli).
 
-Type in `powershell` in the search or run window in Windows. You may see two options - `PowerShell` and `PowerShell ISE`. The first is the regular PowerShell command prompt window. The second is the Interactive Scripting Environment. Feel free to choose either one. Here's [more](http://www.powershellpro.com/powershell-tutorial-introduction/tutorial-windows-powershell-console/) on getting started with PowerShell. Paste and run the following script and you're ready to go. This downloads Chocolatey, sets the environment path variable, and installs the exercism CLI.
+Chocolately lets you install Windows applications quickly and easily. To install Chocolatey please refer to the [installation instructions on the Chocolatey website](https://chocolatey.org/install). 
 
-```powershell
-function which($name) { Get-Command $name -ErrorAction SilentlyContinue | Select-Object Definition }
+> **NOTE**
+> Please ensure that when you open up your Windows shell, whether it's the Command Prompt or Powershell, you open it with **Administrator privileges**, otherwise the Chocolatey installation will fail. If you are unsure how to do this, please refer to the [help below](#open-win-cmd).
 
-if ((which cinst) -eq $null) {
-    iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
-    $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-}
+If you want to check if your Chocolatey installation has been successful, please refer to the [help below](#test-choco-install).
+
+### Installing the Exercism CLI <a name="install-exercism-cli"></a>
+
+With Chocolatey installed, we can now install the Exercism CLI. In your Windows shell, either the Command Prompt or Powershell, type in the following:
 
-cinst exercism-io-cli
 ```
+choco install exercism-io-cli
+```
+> **NOTE**
+>  You don't need to open your shell with Administrator privileges to use the `choco` command.
 
-#### Using the Command Prompt
-1. Open a command line interface (CLI) by clicking on "Start", typing "cmd" into the search bar and pressing enter
-1. Next copy and paste the following command into the command window:
+Type 'Y' to accept any prompts during the installation. If the installation was successful you should have output similar to the following:
 
-        @powershell -NoProfile -ExecutionPolicy unrestricted -Command "iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))" && SET PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin
+![Exercism CLI installation](../../img/win-exercism-installation.png "Exercism CLI installation")
```

Comment:
I don't think the img link thingies are going to work—would you check other markdown files within the `x/docs` directory to see if they've figured out images? I suspect that we'll need to put something in `public/img/...`